<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>232</width>
     <height>88</height>
    </rect>
   </property>
   <property name="title">
    <string>实时数据</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lab_Humi">
      <property name="text">
       <string>DATA1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_Humi"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lab_Temp">
      <property name="text">
       <string>DATA2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_Temp"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>117</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>串口设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="0">
     <widget class="QPushButton" name="btn_search_port">
      <property name="text">
       <string>搜索串口</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_open_port">
      <property name="text">
       <string>打开串口</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>391</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>实时动态曲线显示</string>
   </property>
   <widget class="QCustomPlot" name="widget_plot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
